README.md

```diff
@@ -1 +1,62 @@
-# Steampipe-docs
\ No newline at end of file
+# Steampipe Docs
```

Comment:
Should include a banner image like the rest of our repos - maybe `![image](https://steampipe.io/images/steampipe-social-preview-4.png)` ?